backend/app/migrations/versions/0eee70848f87_add_seed_data_for_git_job_types.py

```diff
@@ -31,7 +31,7 @@
 def upgrade():
     utcnow = datetime.utcnow()
     server_data = [{'id': 3, 'name': 'git-pdf', 'created_at': utcnow, 'updated_at': utcnow},
-                   {'id': 4, 'name': 'git-distribution-preview', 'created_at': utcnow, 'updated_at': utcnow}]
+                   {'id': 4, 'name': 'git-web-hosting-preview', 'created_at': utcnow, 'updated_at': utcnow}]
```

Comment:
We can't change past migrations like this. To change the name, you'd need to add a new migration.